build.gradle

```diff
@@ -285,6 +285,7 @@ def setJavaDocOptions(MinimalJavadocOptions options) {
     options.encoding = 'UTF-8'
     options.charSet 'UTF-8'
     options.docEncoding 'UTF-8'
+    options.addBooleanOption("html4", true)
```

Comment:
Building with JDK 10, show me this warning : 

```
javadoc: warning - You have not specified the version of HTML to use.
The default is currently HTML 4.01, but this will change to HTML5
in a future release. To suppress this warning, please specify the
version of HTML used in your documentation comments and to be
generated by this doclet, using the -html4 or -html5 options.
```

HTML 5 is not working...